docs/navigation/anchor.mdx

```diff
@@ -1 +1,82 @@
-anchor page
\ No newline at end of file
+---
+product: dd360-ds
+title: React Anchor component
+components: Anchor
+---
+
+<br />
+<br />
+
+# <span name="floating-nav">Anchor</span>
+
+With the Anchor component you can create links with customizable properties and support for custom components.
+
+<br />
+
+## <span name="floating-nav">Imports</span>
+
+<WindowEditor codeString="import { Anchor } from 'dd360-ds'" />
+
+## <span name="floating-nav">Usage</span>
+
+<ContainerComponentDoc className="flex items-center">
+    <Anchor to={'/docs/navigation/anchor'}>This is an example</Anchor>
+</ContainerComponentDoc>
+
+<WindowEditor
+    codeString="import { Button, ButtonGroup } from 'dd360-ds'
+
+        <Anchor>This is an example</Anchor> "
+
+/>
+
+<br />
+
+#### <span name="floating-nav">To</span>
+
+With the props to you can redirect to the url you need
```

Comment:
we need wrap the name of the prop to improve the visibility. Like we have on the `getting-started` section

```suggestion
With the prop `to` you can redirect to the url you need
```

And please, remove the word `s` from the props